api/src/main/java/ai/djl/ndarray/NDArray.java

```diff
@@ -2664,6 +2664,17 @@ default NDArray mean(int[] axes) {
      */
     NDArray mean(int[] axes, boolean keepDims);
 
+    /**
+     * Rotates an array by 90 degrees in the plane specified by axes.
+     *
+     * <p>Rotation direction is from the first towards the second axis.
+     *
+     * @param times Number of times the array is rotated by 90 degrees.
+     * @param axes The array is rotated in the plane defined by the axes. Axes must be different.
+     * @return the rotated NDArray
+     */
+    NDArray rot90(int times, int[] axes);
```

Comment:
better use full name, or use:
```
NDArray rotate(int degree, int[] axes)
```

And only support 90/180/270 for now

Reply:
rotate is vague, it essentially just do rotate 90 degree. And numpy use `rot90` as the name. We should keep them consistent.

Reply:
Does java have naming convention for those image transformation function? I am ok with either one